modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -50,6 +102,56 @@ definition {
 		}
 	}
 
+	@description = "LPS-161928 - Verify that it's possible to delete attachments to the notification template"
+	@priority = "3"
+	test CanDeleteAttachment {
+		task ("Creating a object and a attachment field") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161928",
+				objectName = "CustomObject161928",
+				pluralLabelName = "Custom Objects 161928");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Attachment",
+				fieldLabelName = "From Computer And Show Files In DM",
+				fieldName = "fromComputerAndShowFilesInDM",
+				fieldType = "Long",
+				fileSource = "userComputer",
+				isRequired = "false",
+				objectName = "CustomObject161928");
+		}
+
+		task ("Opening the templates and adding infos") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				fromAddress = "test@liferay",
+				fromName = "Test Test",
+				name = "Notification Template");
+		}
+
+		task ("Adding the attachment and saving") {
+			ObjectNotifications.addNotificationTemplateAttachment(objectLabel = "Custom Object 161928");
+
+			Button.clickSave();
+		}
+
+		task ("Going to the Notification Template, deleting the attachment and assert that") {
+			Click(
+				key_objectLabel = "Notification Template",
+				locator1 = "ObjectAdmin#OBJECT_KEBAB_MENU");
+
+			MenuItem.clickNoError(menuItem = "View");
+
+			Click(locator1 = "ObjectNotifications#DELETE_ATTACHMENT_BUTTON");
```

Comment:
Change the task to: `When the attachment is deleted from the notification template`